jenkins/config/oscontainer-secret.yaml

```diff
@@ -0,0 +1,9 @@
+credentials:
+  system:
+    domainCredentials:
+      - credentials:
+        - string:
```

Comment:
Cool with this, though feels like a secret file would be more appropriate for this instead.

Reply:
See https://jenkins-fedora-coreos.apps.ocp.ci.centos.org/configuration-as-code/reference#Credentials-file.

Reply:
I was originally intending to do that, I just couldn't find the syntax for some reason. Thanks for the pointer!

Reply:
It took a bit of trial-and-error. The short version is that you have to base64-encode the file contents to make it work.